pyblish_qml/host.py

```diff
@@ -77,6 +77,22 @@ def _is_headless():
     )
 
 
+def acquire_host_main_window():
+
+    app = QtWidgets.QApplication.instance()
+
+    # Get top window in host
+    _window = app.activeWindow()
+    while True:
```

Comment:
I can see what you're trying to do, but there's an edge-case here that's a little dangerous. Which is that when the active window is a window that isn't parented to Maya (like a third-party, custom window, or poorly designed native window) then this would return that window rather than the main Maya window.

```python
bad_window = QtWidgets.QDialog()
bad_window.show()
# Now this window can be active, and wouldn't return any parent because it has none.
```

Reply:
This one is a bit dangerous indeed !
The alternative I can think of is a piece of code from Avalon
```python
parent = {
    widget.objectName(): widget
    for widget in QtWidgets.QApplication.topLevelWidgets()
}[main_window_name]  # `main_window_name` require further input
```
Will this one better ?

Reply:
Yes, that will work.

`main_window_name` is constant in Maya 2013-2019+ and I don't expect it to change, I would put just `"MayaWindow"`

Reply:
Cool !  I'll try to find the main window name of Nuke and other expensive friends,
or do you have that list ?

Reply:
This seems does not have a quick solution for other DCC Tools :/
Update Maya first !